src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -328,13 +329,25 @@ public function url($path)
      *
      * @param  \League\Flysystem\AwsS3v3\AwsS3Adapter  $adapter
      * @param  string  $path
+     * @param  string  $expires
      * @return string
      */
-    protected function getAwsUrl($adapter, $path)
+    protected function getAwsUrl($adapter, $path, $expires)
```

Comment:
A protected method can be overridden by anybody using Laravel in the world. You cannot know if this is going to break somebody's application or not. This is a breaking change.